src/sdl2_compat.c

```diff
@@ -343,8 +343,10 @@ static QuirkEntryType quirks[] = {
 
 #ifdef __linux__
 static void OS_GetExeName(char *buf, const unsigned maxpath) {
+    int ret;
     buf[0] = '\0';
-    readlink("/proc/self/exe", buf, maxpath);
+    ret = readlink("/proc/self/exe", buf, maxpath);
+    (void)ret;
```

Comment:
Are you getting `unused-result` warnings? If so, is doing `(void) readlink(.....)` not enough?

Reply:
```
sdl2_compat.c:347:11: error: ignoring return value of ‘readlink’ declared with attribute ‘warn_unused_result’ [-Werror=unused-result]
  347 |     (void)readlink("/proc/self/exe", buf, maxpath);

```

I had to patch this way :(
(this should be part of this PR)

Reply:
In any case: if this is really needed, it should be done in a separate patch?